test/remote/gateways/remote_authorize_net_cim_test.rb

```diff
@@ -74,8 +74,17 @@ def test_successful_profile_create_get_update_and_delete
     assert_nil response.authorization
     assert response = @gateway.get_customer_profile(:customer_profile_id => @customer_profile_id)
     assert_nil response.params['profile']['merchant_customer_id']
-    assert_nil response.params['profile']['description']
+    assert_equal 'Up to 255 Characters', response.params['profile']['description']
```

Comment:
Was this a change in the API?

Reply:
I don't know how this ever passed before. The description was being set, so it shouldn't have been asserted as nil.